src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Represents the user interface of the chat bot.
+ * A <code>Ui</code> object corresponds to the user interface of the chat bot.
+ */
+public class Ui {
+    private Scanner scanner;
+
+    /**
+     * Constructs a <code>Ui</code> object.
+     */
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    /**
+     * Reads the next line of input from the user.
+     * @return The next line of input from the user.
+     */
+    public String readCommand() {
+        return scanner.nextLine();
+    }
+
+    /**
+     * Prints the welcome message.
+     */
+    public void printMessage(String message) {
+        System.out.println(message);
+    }
+
+    /**
+     * Prints the welcome message.
+     */
```

Comment:
I liked the naming for these methods! Was wondering if the javadocs for these methods are meant for them?

Reply:
It is not, it was gpt generated :)